### PR TITLE
docs: Add a case when installing Kconfig-frontend

### PR DESCRIPTION
**Pull request by junmin-kim**
Add trouble shooting case when user is using gperf3.1 and
kconfig-frontend-4.11.0.1

### CODE REVIEW

**Comment by seinfra** [comment]
Target : [f5361d133669155b8693f891c760e5e764e09e98] - Code Rule Check (C++) OK.

**Comment by seinfra** [comment]
Target : [f5361d133669155b8693f891c760e5e764e09e98] - Code Rule Check OK.

File: docs/TroubleShooting.md [hunk 1]
```diff
@@ -6,6 +6,38 @@
 
 ## Common
 ### Issues on Kconfig-frontend
+If you are using gperf3.1 and kconfig-frontends-4.11.0.1,
```
Comment (sunghan-chang) [inline docs/TroubleShooting.md]: We had better split two issues by sub-title.
```
#### build break on hconf

#### execution error on mconf
```

Comment (junmin-kim) [reply docs/TroubleShooting.md]: Fixed it

**Comment by seinfra** [comment]
Target : [52b59df3b460dbc4e2f17d29d4af09499d6d88c5] - Code Rule Check (C++) OK.

**Comment by seinfra** [comment]
Target : [52b59df3b460dbc4e2f17d29d4af09499d6d88c5] - Code Rule Check OK.

**Comment by seinfra** [comment]
Target : [ff85970ffb760f72b36cacbfc04008831ed96ede] - Code Rule Check OK.

**Comment by seinfra** [comment]
Target : [ff85970ffb760f72b36cacbfc04008831ed96ede] - Code Rule Check (C++) OK.

File: docs/TroubleShooting.md [hunk 2]
```diff
@@ -6,6 +6,40 @@
 
 ## Common
 ### Issues on Kconfig-frontend
+#### Build break on hconf
+If you are using gperf3.1 and kconfig-frontends-4.11.0.1,
+you may meets below at Kconfig-frontend build(make) time:
```
Comment (sunghan-chang) [inline docs/TroubleShooting.md]: meet

File: docs/TroubleShooting.md [hunk 3]
```diff
@@ -6,6 +6,40 @@
 
 ## Common
 ### Issues on Kconfig-frontend
+#### Build break on hconf
+If you are using gperf3.1 and kconfig-frontends-4.11.0.1,
```
Comment (JeongJunSik) [inline docs/TroubleShooting.md]: How about gperf 3.1 (with space or _ or -)

**Comment by seinfra** [comment]
Target : [ca7ff7369bf473e832c630071ad7dd3191be902d] -

**Comment by seinfra** [comment]
Target : [ca7ff7369bf473e832c630071ad7dd3191be902d] -

**Pull request merged**